code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,22 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(method=INGEST && is_species(M, /datum/species/squid))
+		to_chat(M, "<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
+		M.adjustFireLoss(5, TRUE)
+		sleep(20)
+	if(method=TOUCH && is_species(M, /datum/species/squid))
+		if(M.incapacitated())
+			return
+		var/obj/item/I = M.get_active_held_item()
+		var/list/turf/targets = list()
+		var/turf/T = get_turf(M)
+		for(var/turf/T in oview(M, 3))
+			targets += T
+		if(LAZYLEN(targets) && I)
+			to_chat(M, "<span class='warning'>The salt causes your arm to spasm!</span>")
+			M.log_message("threw [I] due to a Muscle Spasm", LOG_ATTACK)
+			M.throw_item(pick(targets))
```

Comment:
You can simplify all of this with:
`M.throw_item(get_ranged_target_turf(M, pick(GLOB.alldirs), rand(1, 3))`
Then you can toss out the code involving picking a target.

Reply:
Might satisfy Travis also

Reply:
Should I leave get_active_held_item() in here and do a check for it with if(I)?
Don't want to get a warning with no items

Reply:
`throw_item()` already has the same sanity check in it.

Reply:
Oh does it. Well then.